sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample02_DataFeedIngestionOperations.cs

```diff
@@ -89,8 +89,8 @@ public async Task RefreshDataFeedIngestionAsync()
 
             string dataFeedId = DataFeedId;
 
-            var startsOn = DateTimeOffset.Parse("2020-08-01T00:00:00Z");
-            var endsOn = DateTimeOffset.Parse("2020-08-03T00:00:00Z");
+            var startsOn = DateTimeOffset.Parse("2021-06-01T00:00:00Z");
+            var endsOn = DateTimeOffset.Parse("2021-06-03T00:00:00Z");
```

Comment:
New validation by the service. They were complaining that we were trying to re-ingest data from before the ingestion start time we had set up for this data feed. Not gonna lie, they have the right to be mad.